Sources/SKSupport/LineTable.swift

```diff
@@ -11,6 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 import LSPLogging
+import os
```

Comment:
We should guard this by `#if canImport(os)` because the `os` modules is not available on Linux.